facade/service.go

```diff
@@ -720,6 +720,37 @@ func (f *Facade) validateServiceMigration(ctx datastore.Context, svcs []service.
 	return nil
 }
 
+func (f *Facade) RemoveIPs(ctx datastore.Context, args []string) error {
+	serviceID := args[0]
```

Comment:
This call needs to be recorded in the CC audit log. Look through this file for `f.auditLogger` for examples of how to audit log something.